### PR TITLE
NCES public schools data 2020-21 upload

### PR DESCRIPTION
**Pull request by clareconstantine**
This PR adds code to import 2020-2021 NCES data. The files containing the data are on S3. The `dry_run` field is set to `true` so that I can run these manually the first time and make sure they succeed. I'll follow up with a separate PR setting that field to false so this year's data will be seeded automatically in the future.

## Follow-up work

I'll follow up with a PR setting the `dry_run` field to false so that this year's data will be automatically seeded in the future, and there will also be a PR to upload the private school data.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by tim-dot-org — APPROVED** [review]
looks good to me!

File: bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+DRY_RUN = true
+
+VIRTUAL_SCHOOL_MAP = {
+  'Full Virtual' => 'Yes',
+  'Not Virtual' => 'No',
+  'Supplemental Virtual' => 'No',
+  'Virtual with face to face options' => 'Yes',
+  '–' => nil,
+  '†' => nil
```
Comment (bethanyaconnor) [inline bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd:17]: Why do we need these characters in the hashes?

Comment (tim-dot-org) [reply bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd:17]: They're in the raw data as their own values, do you have an example file, clare?

Comment (clareconstantine) [reply bin/oneoff/nces_data/import_school_stats_by_years_2020_2021_ccd:17]: Yes! Here are the files in [drive](https://drive.google.com/drive/u/0/folders/1mNba3MCy3RpyDJDRBI7q1YPDC-cD4v5q) which explain the symbols. (I removed the extra rows at the top and bottom before uploading them to S3)

**Pull request merged**